third_party/terraform/utils/provider_test.go.erb

```diff
@@ -96,6 +106,93 @@ func init() {
 	<% end -%>
 }
 
+var lock = &sync.Mutex{}
```

Comment:
since this variable is package-scoped, can it be called vcrLock or something more descriptive?